server/cluster/cluster.go

```diff
@@ -436,7 +436,7 @@ func (c *RaftCluster) runStoreConfigSync() {
 	for {
 		synced, switchRaftV2Config = c.syncStoreConfig(stores)
 		if switchRaftV2Config {
-			if err := c.opt.Persist(c.GetStorage()); err != nil {
+			if err := c.opt.SwitchRaftV2(c.GetStorage()); err != nil {
```

Comment:
If I manually switch to `v1`, will it switch to `v2` back?

Reply:
In past，the store config will persist also, so it will not switch v1 again after tikv restarted.